src/meta-srv/src/handler.rs

```diff
@@ -458,25 +447,34 @@ pub struct HeartbeatHandlerGroupBuilder {
     region_failure_handler: Option<RegionFailureHandler>,
 
     /// The handler to handle region lease.
-    region_lease_handler: RegionLeaseHandler,
+    region_lease_handler: Option<RegionLeaseHandler>,
 
     /// The plugins.
     plugins: Option<Plugins>,
 
     /// The heartbeat response pushers.
     pushers: Pushers,
+
+    /// The group of heartbeat handlers.
+    handlers: LinkedList<NameCachedHandler>,
```

Comment:
I think Vec is enough as we won't have too many handlers.

Reply:
Resoved in [bcfc11d](https://github.com/GreptimeTeam/greptimedb/pull/4788/commits/bcfc11d465a7212bc4e479ce3fd77e868b77baa7)